crates/accelerate/src/target_transpiler/mod.rs

```diff
@@ -549,19 +534,19 @@ impl Target {
     ///     bool: Returns ``True`` if the instruction is supported and ``False`` if it isn't.
     #[pyo3(
         name = "instruction_supported",
-        signature = (operation_name=None, qargs=None, operation_class=None, parameters=None)
+        signature = (operation_name=None, qargs=Qargs::Global, operation_class=None, parameters=None)
     )]
     pub fn py_instruction_supported(
```

Comment:
The documentation actually says that if `qargs` isn't provided, then the function should return true if it's supported on _any_ qargs, not necessary global.  We might have a testing gap if this default is passing.

We can potentially fix it in a follow-up, if there _is_ a problem, but I think a true fix might be slightly involved and require some assistance from Python space, because we can't use `None` as the sentinel for "not passed".